packages/turf-helpers/index.js

```diff
@@ -23,7 +23,7 @@ export function feature(geometry, properties, bbox, id) {
     if (bbox && bbox.length !== 4) throw new Error('bbox must be an Array of 4 numbers');
     if (id && ['string', 'number'].indexOf(typeof id) === -1) throw new Error('id must be a number or a string');
 
-    var feat = {type: 'Feature'};
+    let feat = {type: 'Feature'};
```

Comment:
Same as comment:
https://github.com/Turfjs/turf/pull/973#discussion_r141083686

Going to merge this PR, we can look into how we want to tackle the ES6 syntax in our final web bundle.